src/main/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/config/AuthAwareTokenConverter.kt

```diff
@@ -12,7 +12,7 @@ class AuthAwareTokenConverter : Converter<Jwt, AbstractAuthenticationToken> {
   private val jwtGrantedAuthoritiesConverter: Converter<Jwt, Collection<GrantedAuthority>> = JwtGrantedAuthoritiesConverter()
 
   companion object {
-    const val SYSTEM: String = "system"
+    const val SYSTEM_USER: String = "system"
```

Comment:
Renamed this const, and refactored uses of a similarly named duplicate const to ensure it is only defined once